StortSpelProjekt/Game/src/main.cpp

```diff
@@ -296,14 +304,19 @@ Scene* GameScene(SceneManager* sm)
 	zombie.model = enemyZombieModel;
 	zombie.hp = 70;
 	zombie.hpBase = 70;
-	zombie.OnHitSounds.emplace_back(L"ZombieHit7");
-    zombie.OnGruntSounds.emplace_back(L"ZombieGnarl1");
-    zombie.OnGruntSounds.emplace_back(L"ZombieGnarl2");
-    zombie.OnGruntSounds.emplace_back(L"ZombieGnarl3");
-    zombie.OnGruntSounds.emplace_back(L"ZombieGnarl4");
-    zombie.OnGruntSounds.emplace_back(L"ZombieGnarl5");
-    zombie.OnGruntSounds.emplace_back(L"ZombieGnarl6");
-    zombie.OnGruntSounds.emplace_back(L"ZombieGnarl7");
+	zombie.onHitSounds.emplace_back(L"ZombieHit7");
+    zombie.onGruntSounds.emplace_back(L"ZombieGnarl1");
+    zombie.onGruntSounds.emplace_back(L"ZombieGnarl2");
+    zombie.onGruntSounds.emplace_back(L"ZombieGnarl3");
+    zombie.onGruntSounds.emplace_back(L"ZombieGnarl4");
+    zombie.onGruntSounds.emplace_back(L"ZombieGnarl5");
+    zombie.onGruntSounds.emplace_back(L"ZombieGnarl6");
+    zombie.onGruntSounds.emplace_back(L"ZombieGnarl7");
+    zombie.onAttackSounds.emplace_back(L"ZombieAttack1");
+    zombie.onAttackSounds.emplace_back(L"ZombieAttack2");
+    zombie.onAttackSounds.emplace_back(L"ZombieAttack3");
+    zombie.onAttackSounds.emplace_back(L"ZombieAttack4");
+    zombie.onAttackSounds.emplace_back(L"ZombieAttack5");
```

Comment:
Might be a mistake from me that has followed since, but the onHit emplacement should not be indented here, if it is intended to be separated from the rest use a newline instead.